app/Task.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = [
+        'search_id',
+        'Sector',
+        'Status',
+        'Group',
+        'Start',
+        'End',
+        'Type',
+        'Description',
+        'Gpx',
+        'GpxFileName',
+        'GpxFile'
```

Comment:
As in all the project all first letter are in lowercase so it should be the same here to folloe the same style